src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -16,8 +16,10 @@ import org.apache.commons.lang3.builder.HashCodeBuilder
   * @param processorJoint Parameter that identifies joint as being artificial joint introduced as part
   *                        of load balancing
   */
+@SerialVersionUID(1L)
 case class Face(normalVec: Array[Double], distance: Double, phi: Double, cohesion: Double,
-                processorJoint: Boolean=false) {
+                processorJoint: Boolean=false)
+extends Serializable {
```

Comment:
This is weird formatting to have the `extends` statement down on its own line here. You can just put it right after the end of the constructor parameter list, like this:
```scala
case class Face(....) extends Serializable {
```